src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+function createNotification(messange, type) {
```

Comment:
There is a typo in the parameter name `messange`. It should be `message` to correctly reflect its purpose.